src/migrations/1761808658016-url_analytics.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UrlAnalytics1761808658016 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE IF NOT EXISTS "url_analytics" (
+           "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),       
+           "url_id" uuid ,
```

Comment:
```suggestion
           "url_id" uuid NOT NULL ,
```

Also check for other columns as well. by default all the columns are null, we need to specific the columns that are not null